### DISCUSSION TITLE
Are sounds only available through Wwaves?

### DISCUSSION THREAD

**Discussion by nikitanyc** [flybywiresim/aircraft#2863] [Q&A] [answered]
I would like to map some .wav sounds to switches/buttons but it seems like they are all Wwaves. I'm unfamiliar with .wem. Is there any way to map .wav to certain objects?

**Comment by Inaxair** [comment]
I don't know if it's possible, but we use the Wwise technology as it's the best for now and really good one. There might have converters on the internet though.

Thanks!
Inaxair from FlyByWire Simulations

**Comment by hotshotp** [comment, marked as answer]
To change sounds in the mod you will have to learn how to use wwise which is a complex software and audio engine that is for audio integration into the sim. Please do not map sounds to switches and buttons because we have recorded every button from a real a320 and are replacing the default ones with them - we haven't replaced everything including any of the overhead pushbuttons yet and we are working on it.
